lib/units/include/units.h

```diff
@@ -0,0 +1,5140 @@
+//--------------------------------------------------------------------------------------------------
```

Comment:
Can you move this file under lib/utility/include instead? Also make sure to update the CMakeLists.txt under lib/utility to include this